### ISSUE TITLE
Failed to run make libbiometry_fp_api_32

### ISSUE THREAD

**Issue by 0312birdzhang** [piggz/sailfish-fpd-community#16]
```
hybris/mw/sailfish-fpd-community/android/hybris/biometry_fp_hidl_for_hybris.cpp:18:10: fatal error: 'biometry/hardware/biometry.h' file not found
#include <biometry/hardware/biometry.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[ 32% 19/58] //hardware/interfaces/biometrics/fingerprint/2.1:android.hardware.biometrics.fingerprint@2.1 clang++ android/hardware/biometrics/fingerprint/2.1/BiometricsFingerprintAll.cpp [arm]
ninja: build stopped: subcommand failed.
10:17:19 ninja failed with: exit status 1
```

After manually copy [biometryd/include](https://github.com/piggz/biometryd/tree/xenial/include) dir to sailfish-fpd-community, build pass.

**Comment by rinigus** [comment]
That's fixed in #12, wait for PR to get merged

**Comment by Verevka** [comment]
https://github.com/SailfishOS-sagit/android_device_xiaomi_msm8998-common/commit/739b4affcca5c52590bcbe6a915282864d9aa239
